includes/abstracts/abstract-wc-stripe-payment-gateway.php

```
@@ -1616,9 +1616,22 @@ function ( $item ) use ( $currency, $order_id ) {
-				$unit_cost       = WC_Stripe_Helper::get_stripe_amount( ( $subtotal / $quantity ), $currency );
-				$tax_amount      = WC_Stripe_Helper::get_stripe_amount( $item->get_total_tax(), $currency );
-				$discount_amount = WC_Stripe_Helper::get_stripe_amount( $subtotal - $item->get_total(), $currency );
+				$total_tax = $item->get_total_tax();
+
+				if ( $subtotal >= 0 ) {
+					$unit_cost       = WC_Stripe_Helper::get_stripe_amount( ( $subtotal / $quantity ), $currency );
+					$discount_amount = WC_Stripe_Helper::get_stripe_amount( $subtotal - $item->get_total(), $currency );
+				} else {
+					$unit_cost       = 0;
+					$discount_amount = WC_Stripe_Helper::get_stripe_amount( ( $subtotal / $quantity ), $currency );
+				}
```

PHPMD (2.15.0)

[error] The method get_level3_data_from_order uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)